### PR TITLE
Issue7374

### PR DESCRIPTION
**Pull request by alejomoreira**

### CODE REVIEW

File: tests/Issues/Issue7374/Fixture/fixture.php.inc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rector\Core\Tests\Issues\7374\Fixture;
```
Comment (samsonasik) [inline tests/Issues/Issue7374/Fixture/fixture.php.inc]: namespace is invalid, should be: 

```suggestion
namespace Rector\Core\Tests\Issues7374\Fixture;

File: tests/Issues/Issue7374/error.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+namespace Rector\Core\Tests\Issues\Issue7374;
+
+use Iterator;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+/**
+ * @see https://github.com/rectorphp/rector/issues/7374
+ */
+final class RuleCombinationShouldNotReturnGetAttributeOnNullReturnTest extends AbstractRectorTestCase
```
Comment (samsonasik) [inline tests/Issues/Issue7374/error.php:14]: File name need to match with class name: , so `error.php` should be `RuleCombinationShouldNotReturnGetAttributeOnNullReturnTest.php`

**Comment by TomasVotruba** [comment]
Just for reference so we could find it, test case for https://github.com/rectorphp/rector/issues/7374

**Comment by TomasVotruba** [comment]
Thank you for the test fixture @alejomoreira :+1: 

I'm looking into this...

**Comment by TomasVotruba** [comment]
I've cherry picked the test case here: https://github.com/rectorphp/rector-src/pull/2770
Yet I got different error there then getAttribute on null originaly reported in https://github.com/rectorphp/rector/issues/7374

Let's merge it and see if that helps your issue :slightly_smiling_face:

**Pull request closed without merging**